css-layout-classes/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="styles.css">
+  <title>CSS Layout Classes</title>
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <header>
+          <h1>Choose your character!</h1>
+        </header>
+      </div>
+    </div>
+    <div class="column-full">
+      <article>
+        <div class="row">
+          <div class="column-one-third">
+            <img src="./images/link.png">
+          </div>
+          <div class="column-one-third">
+            <h3>Super Smash Bros. Ultimate</h3>
+            <p>Super Smash Bros. Ultimate received universal acclaim from both critics and players, with some critics calling it the best installment in the series.
+              It received praise for its large amount of content and fine-tuning of existing Smash gameplay elements, although its online mode was widely criticized.
+            </p>
+          </div>
+          <div class="column-one-third">
+            <img src="./images/yoshi.png">
+          </div>
+        </div>
+      </article>
+    </div>
+    <div class="column-full">
+      <article>
```

Comment:
I would remove the column full and article elements here, you just want the `row` class with the four column one four elements